amazon-msk/cdk/web-streaming/cdk.json

```diff
@@ -0,0 +1,89 @@
+{
+  "app": "npx ts-node --prefer-ts-exts bin/web-streaming.ts",
+  "watch": {
+    "include": [
+      "**"
+    ],
+    "exclude": [
+      "README.md",
+      "cdk*.json",
+      "**/*.d.ts",
+      "**/*.js",
+      "tsconfig.json",
+      "package*.json",
+      "yarn.lock",
+      "node_modules",
+      "test"
+    ]
+  },
+  "context": {
+    "vpcId": "<your VPC ID>",
+    "msk":
+    {
+      "bootstrapServers": "<Bootstrap Servers of your MSK cluster>",
+      "credentialsSecretName": "<Secret Name associated with your MSK cluster>"  
+    },
+    "publicTlsCertificateKey": "<your public tls certificate key ARN>",
+    "kafkaTopic": "<your kafka topic to be exposed on HTTP and SSE>",
+    "@aws-cdk/aws-lambda:recognizeLayerVersion": true,
+    "@aws-cdk/core:checkSecretUsage": true,
+    "@aws-cdk/core:target-partitions": [
+      "aws",
+      "aws-cn"
+    ],
```

Comment:
Perhaps the full set of parameters should be inside something like `"zilla-plus"` under `"context"`?

Reply:
I like that!